ui/editor/src/ctrl.ts

```diff
@@ -190,6 +190,11 @@ export default class EditorCtrl {
     return `${this.cfg.baseUrl}/${urlFen(fen)}${variant}${orientationParam}`;
   }
 
+  makeImageUrl(fen: string, orientation: Color = 'white'): string {
+    const url = `https://lichess1.org/export/fen.gif?fen=${fen}&color=${orientation}`;
```

Comment:
Can you url encode the FEN string? either with underscores like the URL above it has or urlencoding.